examples/requirements.txt

```diff
@@ -6,3 +6,4 @@ streamlit==1.22.0
 tornado==6.3.1
 uvicorn==0.21.0
 Werkzeug==2.2.2
+Hypercorn==0.16.0
```

Comment:
nit: could we keep in alpha order?

Reply:
Looks like you added to the base requirements, but didn't change this to alpha order :)